main.cc

```diff
@@ -1704,6 +1705,12 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 mm_notifier.local().unregister_listener(&ss.local()).get();
             });
 
+            supervisor::notify("starting the view builder");
```

Comment:
If I understand this change correctly, this message now appears at the wrong place - it appears when creating some silly sharded variable - not when really running the start function below (inside the if cfg->view_building)

Reply:
Will check, thanks

Reply:
It's now symmetrical with view update generator start:

```c++
            supervisor::notify("starting view update generator");
            view_update_generator.start(std::ref(db), std::ref(proxy), std::ref(stop_signal.as_sharded_abort_source())).get();
            auto stop_view_update_generator = defer_verbose_shutdown("view update generator", [] {
                view_update_generator.stop().get();
            });
```

and the v.u.g.::start() itself is invoked later under `if cfg->view_building` check